fastlane/swift/MainProcess.swift

```diff
@@ -0,0 +1,62 @@
+//
+//  MainProcess.swift
+//  FastlaneSwiftRunner
+//
+//  Created by Joshua Liebowitz on 8/26/17.
+//
+
+//
+//  ** NOTE **
+//  This file is provided by fastlane and WILL be overwritten in future updates
+//  If you want to add extra functionality to this project, create a new file in a
+//  new group so that it won't be marked for upgrade
+//
```

Comment:
This header and the 3 lines at the bottom make it look like this file is part of the Swift code generator, although I don't think they were actually auto generated. Maybe cleaning up those files and adding some doc comments would be great to instruct future readers/maintainers how to handle this file 👍